### DISCUSSION TITLE
Need help with GitHub Education Student Developer Pack Benefits

### DISCUSSION THREAD

**Discussion by alisajad001** [community/community#116863] [GitHub Education] [answered]
Hello everyone, 
it's been four days since I received an approval but I don't have access to my benefits. 

What should I do?

![20240405_083333.jpg](https://github.com/community/community/assets/126584124/442e11f0-eda1-4520-82bc-e099ef933c8c)

**Comment by alisajad001** [comment, marked as answer]
> Hello everyone, 
> it's been four days since I received an approval but I don't have access to my benefits. 
> 
> What should I do?
> 
> ![20240405_083333.jpg](https://github.com/community/community/assets/126584124/442e11f0-eda1-4520-82bc-e099ef933c8c)
> 
> 

Update:
Just got the benefits today.
Gonna start a new journey 😀.
Thank you so much GitHub.

**Reply by boules131** [reply]
![Screenshot 2024-04-05 141141](https://github.com/community/community/assets/135156301/96dc2322-7df7-4735-a87e-7a67031a5691)
i need help 😭

**Reply by alisajad001** [reply]
> ![Screenshot 2024-04-05 141141](https://github.com/community/community/assets/135156301/96dc2322-7df7-4735-a87e-7a67031a5691)
> i need help 😭

You may have to submit your student card with your Desktop Camera

**Reply by boules131** [reply]
thanks

**Reply by severoleonardo** [reply]
Very good, @alisajad001. Don't forget to take advantage and explore the benefits to the fullest. Many of the tools available on GitHub Education are used by companies today and it is an excellent opportunity to prepare and stay up to date.

Some essential tools that I recommend regardless of your Stack:
- GitHub Copilot
- Namecheap
- .TECH
-Microsoft Azure
- GitLens
- GitKraken
- DeepSource
- Datadog
- AlgoExpert
- Interview Cake
- Tower
- and much more...

**Extra tip: GitHub is always adding and removing tools from bundles. So don't take too long to test them!**

**Reply by alisajad001** [reply]
Thank you so much. I'm gonna try all of them

**Comment by FAIZUAN55** [comment]
Sabar

Pada Jum, 5 Apr 2024, 8:15 PTG Boules Ashraf Samir Sobhy <
***@***.***> menulis:

**Comment by samus-aran** [comment]
Just as a reminder to everyone! When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. **Your educational benefits will arrive within 72 hours of this approval. However, during busy periods like the start of a new semester, it may take a bit longer.**

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.

**Comment by Vundla** [comment]
hello i recieved an approved but have no access to my benefits

**Reply by alisajad001** [reply]
Hey you need to wait at least three days to get the full benefits.
